index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>판다마켓</title>
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.6/dist/web/static/pretendard.css"
+   >
+    <link rel="stylesheet" href="style/style.css">
+</head>
+<body>
+    <header>
+        <div class="wrap">
+            <a href ="/">
+                <img class="logo" src="./img/logo/logo.png" width="153">
```

Comment:
P3:
이미지 태그를 쓰실때는 alt 속성을 같이 기입해주시면 좋겠습니다.

https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt#usage_notes

```suggestion
                <img class="logo" src="./img/logo/logo.png" width="153" alt='판다마켓'>
```